namespace-specific-policy/README.md

```diff
@@ -42,20 +42,33 @@ The folder directory of the `namespace-specific-policy` example:
 │   ├── kustomization.yaml
 │   └── README.md
 ├── configsync-src
-│   ├── base
-│   │   ├── kustomization.yaml
-│   │   ├── namespace.yaml
-│   │   ├── networkpolicy.yaml
-│   │   ├── rolebinding.yaml
-│   │   └── role.yaml
-│   ├── team-a
-│   │   └── kustomization.yaml
-│   ├── team-b
-│   │   └── kustomization.yaml
-│   └── team-c
-│       └── kustomization.yaml
+│   ├── example
+│   │   ├── base
+│   │   │   ├── kustomization.yaml
+│   │   │   ├── namespace.yaml
+│   │   │   ├── networkpolicy.yaml
+│   │   │   ├── rolebinding.yaml
+│   │   │   └── role.yaml
+│   │   ├── team-a
+│   │   │   └── kustomization.yaml
+│   │   ├── team-b
+│   │   │   └── kustomization.yaml
+│   │   └── team-c
+│   │       └── kustomization.yaml
+│   ├── external-team
+│   │   ├── kustomization.yaml
+│   │   ├── namespace.yaml
+│   │   ├── networkpolicy.yaml
+│   │   ├── rolebinding.yaml
+│   │   └── role.yaml
```

Comment:
The resources in `external-team` folder should be deleted.

Reply:
Done